package.json

```diff
@@ -4,24 +4,19 @@
   "description": "ESLint rules according to kentcdodds' personal preference",
   "main": "index.js",
   "scripts": {
-    "lint": "eslint .",
-    "test": "npm-run-all --parallel lint find-new-rules",
     "find-new-rules": "npm-run-all --parallel find-new-rules:*",
-    "find-new-rules:main": "eslint-find-rules --unused ./index.js",
-    "find-new-rules:jsx-a11y": "eslint-find-rules --unused ./other/test/jsx-a11y/index.js",
     "find-new-rules:jest": "eslint-find-rules --unused ./other/test/jest/index.js",
+    "find-new-rules:jsx-a11y": "eslint-find-rules --unused ./other/test/jsx-a11y/index.js",
+    "find-new-rules:main": "eslint-find-rules --unused ./index.js",
     "find-new-rules:react": "eslint-find-rules --unused ./other/test/react/index.js",
     "find-new-rules:webpack": "eslint-find-rules --unused ./other/test/webpack/index.js",
-    "validate": "npm t"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "pretty-quick --staged && npm run validate"
-    }
+    "lint": "kcd-scripts lint",
```

Comment:
Oh, I want to use this project's lint config to lint itself as an additional test.

Reply:
I've set the linting config to `./index.js`, so this project will lint itself

Reply:
Oh. Missed that.